jetty-server/src/main/java/org/eclipse/jetty/server/session/SessionCache.java

```diff
@@ -118,6 +118,16 @@ default Session renewSessionId(String oldId, String newId, String oldExtendedId,
     {
         return renewSessionId(oldId, newId);
     }
+    
+    /**
+     * Adds a new Session, with a never-before-used id,
+     *  to the cache.
+     * 
+     * @param id
+     * @param session
+     * @throws Exception
+     */
+    void add(String id, Session session) throws Exception;
```

Comment:
Isn't this a breaking change? `SessionCache` seems to be an interface that people would implement, no?